base/applications/notepad/lang/sl-SI.rc

```diff
@@ -44,101 +44,101 @@ BEGIN
         MENUITEM "&Prilepi\tCtrl+V", CMD_PASTE
         MENUITEM "Izbri&ši\tDel", CMD_DELETE
         MENUITEM SEPARATOR
-        MENUITEM "&Najdi ...", CMD_SEARCH
+        MENUITEM "P&oišči ...", CMD_SEARCH
         MENUITEM "Na&daljuj iskanje\tF3", CMD_SEARCH_NEXT
-        MENUITEM "Replace...\tCtrl+H", CMD_REPLACE
-        MENUITEM "Go To...\tCtrl+G", CMD_GOTO
+        MENUITEM "Zamenjaj...\tCtrl+H", CMD_REPLACE
```

Comment:
Please fix the inconsistency of spacing. In Slovenian, unless a word is cut short, the ellipsis mark needs a space before it (nestično je).

Reply:
still a problem